### PR TITLE
docs(datepicker): add demo for bsConfig

### PR DESCRIPTION
**Pull request by ghost**
add demo for bsConfig related to #3632
In this pr I am checking work of bsConfig property. Adding bsConfig property http://joxi.ru/LmG9e3QFRPlqPA you can manage style and format of the datepicker. In this example I have change style of date output and set color of the datepicker theme.http://joxi.ru/E2pE1k5tBLGn3A

### CODE REVIEW

File: demo/src/app/components/+datepicker/datepicker-section.list.ts [hunk 1]
```diff
@@ -131,6 +132,14 @@ export const demoComponentContent: ContentSection[] = [
         component: require('!!raw-loader?lang=typescript!./demos/reactive-forms/reactive-forms.component.ts'),
         html: require('!!raw-loader?lang=markup!./demos/reactive-forms/reactive-forms.component.html'),
         outlet: DemoDatepickerReactiveFormsComponent
+      },
+      {
+        title: 'Config properties',
+        anchor: 'config-object',
+        component: require('!!raw-loader?lang=typescript!./demos/config-object/config-object.ts'),
+        html: require('!!raw-loader?lang=markup!./demos/config-object/config-object.html'),
+        description: `<p>You can set options for datepicker config object</p>`,
```
Comment (IlyaSurmay) [inline demo/src/app/components/+datepicker/datepicker-section.list.ts]: You can configure the datepicker via its `bsConfig` option

**Review by IlyaSurmay — CHANGES_REQUESTED** [review]

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
PLease provide more detailed description - what should be displayed in demo? What kind of property/possibilities etc. it shows?

**Comment by ghost** [comment]
update and testing my branch according to Yevheniia comments(conflicts in files) which I can't see

**Review by IlyaSurmay — DISMISSED** [review]

**Comment by YevheniiaMazur** [comment]
Tested, looks good

**Review by valorkin — DISMISSED** [review]

**Pull request merged**